CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Host/Advanced] Add the field `Scheduler granularity`. Only available on XCP-ng >= 8.2 and CH >= 8.2 [#5191](https://github.com/vatesfr/xen-orchestra/issues/5291) (PR [#5320](https://github.com/vatesfr/xen-orchestra/pull/5320))
```

Comment:
Better specify what functionality we add.

```suggestion
- [Host/Advanced] Ability to update the scheduler granularity. Only available on XCP-ng >= 8.2 and XS >= 8.2 [#5191](https://github.com/vatesfr/xen-orchestra/issues/5291) (PR [#5320](https://github.com/vatesfr/xen-orchestra/pull/5320))
```

Reply:
XS is CH since 8.0, though.
```suggestion
- [Host/Advanced] Ability to change the scheduler granularity. Only available on XCP-ng >= 8.2 and CH >= 8.2 [#5191](https://github.com/vatesfr/xen-orchestra/issues/5291) (PR [#5320](https://github.com/vatesfr/xen-orchestra/pull/5320))
```